shopfloor/tests/test_zone_picking_start.py

```diff
@@ -30,6 +30,14 @@ def setUpClassBaseData(cls, *args, **kwargs):
         cls._update_qty_in_location(cls.zone_sublocation1, cls.product_b, 10)
         extra_picking.action_assign()
 
+    def test_data_for_zone(self):
+        pass
```

Comment:
TODO or not TODO?

Reply:
ops, forgot about it... yes!

Reply:
I don't have time now to add this test. I'll keep the PR opened and get back to it ASAP.

Reply:
test added!